autoload/tagbar.vim

```diff
@@ -2320,22 +2326,26 @@ function! s:JumpToTag(stay_in_tagbar) abort
         let taginfo.fileinfo.fline[curline] = taginfo
     endif
 
-    " Center the tag in the window and jump to the correct column if
-    " available, otherwise try to find it in the line
-    normal! z.
+    if g:tagbar_jump_lazy_scroll != 0 && alreadyvisible
```

Comment:
Change this to call the fore mentioned function (hence only running if the option is set).

Reply:
See comment above for why we have to call the function above and cannot call it in the `if` statement.